src/index.html

```diff
@@ -6,9 +6,57 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img class="header-logo" src="images/logo.png"
+             alt="logoo`">
+      </a>
+      <nav class="header-navigation">
+        <ul class="header-navigation__items">
+<<<<<<< HEAD
+          <li class="header-navigation__item">
+            <a href="#apple" class="header-navigation__links is-active">Apple</a>
+=======
+          <li class="header-navigation__item header-navigation__item--active">
+            <a href="#apple" class="header-navigation_links">Apple</a>
+>>>>>>> 86bdc12aa3b103cc4b57244a64d0a090bca52438
```

Comment:
please remove unnecessary stuff from code and fix the html
```suggestion
          <li class="header-navigation__item">
            <a href="#apple" class="header-navigation__links is-active">Apple</a>
          <li class="header-navigation__item header-navigation__item--active">
            <a href="#apple" class="header-navigation_links">Apple</a>
```